lab_2_keywords_cooccurrence/main.py

```diff
@@ -69,9 +127,23 @@ def calculate_word_scores(word_degrees: Mapping[str, int],
 
     In case of corrupt input arguments, None is returned
     """
-    pass
+    if not isinstance(word_degrees, dict) or word_degrees == {}:
```

Comment:
this is a bad way to check whether the object is empty
consider doing it as `if not word_degrees`

Reply:
it applies to all checks of this kind